packages/grafana-ui/src/components/ThresholdsEditorNew/ThresholdsEditor.tsx

```diff
@@ -38,15 +38,22 @@ interface State {
 }
 
 export class ThresholdsEditor extends PureComponent<Props, State> {
+  private latestThresholdInputRef: HTMLInputElement | null;
```

Comment:
One last question - why are you not using `React.createRef` for this?

Reply:
Hmm... I think I read the react docs a little too literally around when to use callback refs. I've gone back to using `createRef`. 👍